terraform/storage/provision/outputs.tf

```diff
@@ -1,2 +1,6 @@
 output "id" { value = google_storage_bucket.bucket.id }
 output "bucket_name" { value = var.name }
+output "status" { value = format("created bucket %s - URL: https://console.cloud.google.com/storage/browser/%s;tab=objects?project=%s",
```

Comment:
suggestion: we can use the sequence specifier in order to avoid specifying the same argument multiple times. See: https://developer.hashicorp.com/terraform/language/functions/format#specification-syntax

Reply:
I love it!! I'll change it. Thank you very much @jhvhs